flow/layers/backdrop_filter_layer.cc

```diff
@@ -26,11 +26,11 @@ void BackdropFilterLayer::Diff(DiffContext* context, const Layer* old_layer) {
   context->AddLayerBounds(paint_bounds);
 
   if (filter_) {
-    context->GetTransform().mapRect(&paint_bounds);
+    context->GetTransform3x3().mapRect(&paint_bounds);
```

Comment:
Doesn't DiffContext have a MapRect method to avoid getting the Transform3x3 (twice since you also get it down below as well)?